spec/scoreboard_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'scoreboard'
+describe Scoreboard do
+  let(:scoreboard) do
+    Scoreboard.new([
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0],
+                     [0, 0]
+                   ])
+  end
+
+  let(:scoreboard_1) do
+    Scoreboard.new([
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10],
+                     [10, 10, 10]
+                   ])
+  end
+  describe '#initialize' do
+    it 'initialize an instance of Scoreboard' do
+      expect(scoreboard).to be_a Scoreboard
+    end
+  end
+
+  describe '.result' do
+    it 'should have result attribute' do
+      expect(scoreboard).to have_attributes(result: 0)
+    end
+  end
+
+  describe '.result' do
+    it 'returns lowest score of game as 0' do
+      expect(scoreboard.result).to eq 0
+    end
+    it 'returns maximum score of game as 300' do
+      expect(scoreboard_1.result).to eq 300
+    end
+  end
+  describe '#calculate_score' do
+    it 'should calculate_score' do
+      expect(scoreboard_1).to respond_to(:calculate_score)
+    end
```

Comment:
It seems like calculate score acts as a private or helper function in your class, not really a function that should be called directly by the user. In fact, in all the following tests you are testing the result, not calling calculate_scores directly.
I think you should remove this test and put all the other ones in the `describe .result` as they are using `.result`